### PR TITLE
Add wiki article for `CWC 2022`

### PR DESCRIPTION
**Pull request by LeoFLT**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/CWC/2022/en.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+| Quarterfinals | 2022-06-11/2022-06-12 |
+| Semifinals | 2022-06-18/2022-06-19 |
+| Finals | 2022-06-25/2022-06-26 |
+| Grand Finals | 2022-07-02/2022-07-03 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $150 per team member, unique profile badge, *osu!catch Champion* user title for one year |
```
Comment (Walavouchey) [inline wiki/Tournaments/CWC/2022/en.md:31]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $150 per team member, unique profile badge, **osu!catch Champion** user title for one year |
```

Comment (peppy) [inline wiki/Tournaments/CWC/2022/en.md:31]: I thought we changed the way prizes were listed to not show the per-member prizes (which makes the pool look much smaller than it actually is).. am I imagining things, or is this using an old format?

Comment (Walavouchey) [reply wiki/Tournaments/CWC/2022/en.md]: Almost all tournaments with prize pools display them this way, at least from the tournament PRs I've looked at. From looking, it seems like OWC [2020](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2020) & [2021](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2021) and a few other tournaments display the whole amount

<details><summary> output of grep -FRn "prize pool" in the tournament folder</summary>

```
CIC/2014/en.md:29:*Note: Still looking for confirmation of prize pool.*
CIC/2018/en.md:29:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the prize pool, profile badge |
CIC/2018/en.md:30:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the prize pool |
CIC/2018/en.md:31:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the prize pool |
CIC/2019/en.md:31:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, 1 year of Discord Nitro subscription, unique profile badge |
CIC/2019/en.md:32:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool |
CIC/2019/en.md:33:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool |
CIC/2020/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | IDR 1.000.000,- + 50% of the raised prize pool, 3 months of osu!supporter, unique profile badge |
CIC/2020/en.md:31:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | IDR 600.000,- + 30% of the raised prize pool, 2 months of osu!supporter |
CIC/2020/en.md:32:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | IDR 400.000,- + 20% of the raised prize pool, 1 month of osu!supporter |
CIC/2021/en.md:29:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, 3 months of osu!supporter, unique profile badge |
CIC/2021/en.md:30:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool, 2 months of osu!supporter |
CIC/2021/en.md:31:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool, 1 month of osu!supporter |
CIC/2022/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st Place") | 50% of the raised prize pool, 6 months of osu!supporter, unique profile badge |
CIC/2022/en.md:31:| ![Silver crown](/wiki/shared/crown-silver.png "2nd Place") | 30% of the raised prize pool, 4 months of osu!supporter |
CIC/2022/en.md:32:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd Place") | 20% of the raised prize pool, 2 month of osu!supporter |
CMC/2020/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 3 months of osu!supporter for each team member, 100% of the raised prize pool |
CWC/2016/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge, special user title |
CWC/2016/en.md:31:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 38% of the raised prize pool, unique profile badge |
CWC/2016/en.md:32:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 12% of the raised prize pool, unique profile badge |
GTS/CGTS_2021/en.md:57:A prize pool of $1,000 with physical prizes are made possible thanks to [Lotuspro](http://www.lotuspro.io/).
ICC/2020/en.md:29:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | IDR 1.000.000, ~50% of the raised prize pool, unique profile badge |
ICC/2020/en.md:30:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | IDR 600.000, ~30% of the raised prize pool |
ICC/2020/en.md:31:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | IDR 450.000, ~20% of the raised prize pool |
ICC/2021/en.md:29:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge |
ICC/2021/en.md:30:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool |
ICC/2021/en.md:31:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool |
MP/2019_4K/en.md:31:The prize pool was crowdfunded through the [tipanddonation](https://tipanddonation.com/MP4K3) page with a minimum prize pool set at 200 PLN total with a 50%/30%/20% split for the top 3 players.
MWC/2016_4K/en.md:30:There was a $1,900 cash prize pool with extra money being added by buying support banners to show support for their teams!
MWC/2016_4K/en.md:34:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu!mania Champion" user title for one year |
MWC/2016_4K/en.md:35:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 38% of the raised prize pool, unique profile badge |
MWC/2016_4K/en.md:36:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 12% of the raised prize pool, unique profile badge |
MWC/2016_7K/en.md:30:There was a $1,900 cash prize pool with extra money being added by buying support banners to show support for their teams!
MWC/2016_7K/en.md:34:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu!mania Champion" user title for one year |
MWC/2016_7K/en.md:35:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 38% of the raised prize pool, unique profile badge |
MWC/2016_7K/en.md:36:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 12% of the raised prize pool, unique profile badge |
OIT/2020/en.md:31:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, 6 months of osu!supporter, unique profile badge |
OIT/2020/en.md:32:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool, 4 months of osu!supporter |
OIT/2020/en.md:33:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool, 2 months of osu!supporter |
OIT/2021/en.md:31:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, 6 months of osu!supporter, unique profile badge |
OIT/2021/en.md:32:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool, 4 months of osu!supporter |
OIT/2021/en.md:33:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool, 2 months of osu!supporter |
OIT/2022/en.md:31:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2,000,000 IDR + 50% of the additional raised prize pool, physical trophy (courtesy of [Herculex Indonesia](https://www.tokopedia.com/herculex)), unique profile badge |
OIT/2022/en.md:32:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1,000,000 IDR + 30% of the additional raised prize pool, physical trophy (courtesy of [Herculex Indonesia](https://www.tokopedia.com/herculex)) |
OIT/2022/en.md:33:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 500,000 IDR + 20% of the additional raised prize pool, physical trophy (courtesy of [Herculex Indonesia](https://www.tokopedia.com/herculex)) |
OWC/2015/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu! Champion" user title |
OWC/2015/en.md:31:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 38% of the raised prize pool, unique profile badge |
OWC/2015/en.md:32:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 12% of the raised prize pool, unique profile badge |
OWC/2020/en.md:29:The osu! World Cup 2020 offered a $5,000 cash prize pool and limited edition merch!
OWC/2020/en.md:33:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the prize pool, exclusive single-run merch, profile badge, "osu! Champion" user title for one year |
OWC/2020/en.md:34:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 36% of the prize pool, exclusive single-run merch, profile badge |
OWC/2020/en.md:35:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 14% of the prize pool, exclusive single-run merch, profile badge |
OWC/2021/en.md:29:The osu! World Cup 2021 offered a $5,000 cash prize pool and limited edition merch!
OWC/2021/en.md:33:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 48% of the prize pool, exclusive single-run merch, profile badge, "osu! Champion" user title for one year |
OWC/2021/en.md:34:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 32% of the prize pool, exclusive single-run merch, profile badge |
OWC/2021/en.md:35:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the prize pool, exclusive single-run merch, profile badge |
SOFT/5/en.md:36:The Springtime Osu!mania Free-for-all Tournament 5 offered a $250 minimum cash prize pool!
SOFT/5/en.md:40:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the prize pool, profile badge |
SOFT/5/en.md:41:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 34% of the prize pool, profile badge |
SOFT/5/en.md:42:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 16% of the prize pool, profile badge |
SOFT/6/en.md:35:The Springtime osu!mania Free-for-all Tournament 6 will offer a $250 minimum cash prize pool!
SOFT/6/en.md:39:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 40% of the prize pool, profile badge |
SOFT/6/en.md:40:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 25% of the prize pool, profile badge |
SOFT/6/en.md:41:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the prize pool, profile badge |
SOFT/6/en.md:42:| 4th place | 10% of the prize pool |
SOFT/6/en.md:43:| 5th-6th place | 5% of the prize pool |
SST/2018/en.md:22:The SS Tournament 2018 had a prize pool of USD 200.
SST/2018/en.md:26:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of total prize pool, unique profile badge |
SST/2018/en.md:27:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of total prize pool |
SST/2018/en.md:28:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of total prize pool |
TWC/2016/en.md:30:| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, unique profile badge, "osu!taiko Champion" user title for one year |
TWC/2016/en.md:31:| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 38% of the raised prize pool, unique profile badge |
TWC/2016/en.md:32:| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 12% of the raised prize pool, unique profile badge |
VAF/1/en.md:36:A prize pool of grand total 18 months of osu!supporter and physical trophies is made possible thanks to [Lotuspro](http://www.lotuspro.io/).
```
</details>

Comment (LeoFLT) [reply wiki/Tournaments/CWC/2022/en.md]: Updated to 48%/32%/20% for 1st/2nd/3rd places and changed the total to prize pool to $2000 as discussed.

**Review by Walavouchey — APPROVED** [review]

File: wiki/Tournaments/CWC/2022/en.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+| Round of 32 | 2022-05-28/2022-05-29 |
+| Round of 16 | 2022-06-04/2022-06-05 |
+| Quarterfinals | 2022-06-11/2022-06-12 |
+| Semifinals | 2022-06-18/2022-06-19 |
+| Finals | 2022-06-25/2022-06-26 |
+| Grand Finals | 2022-07-02/2022-07-03 |
+
+## Prizes
+
+The osu!catch Cup 2022 offers a $2,000 cash prize pool.
```
Comment (Walavouchey) [inline wiki/Tournaments/CWC/2022/en.md:29]:
```suggestion
The osu!catch World Cup 2022 offers a $2,000 cash prize pool.
```

File: wiki/Tournaments/CWC/2022/en.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+The **osu!catch World Cup 2022** (***CWC 2022***) is a country-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It is the eleventh instalment of the osu!catch World Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-04-21/2022-05-05 |
+| Qualifier showcase | 2022-05-14 (14:00 UTC) |
+| Qualifier stage | 2022-05-21/2022-05-22 |
```
Comment (Walavouchey) [inline wiki/Tournaments/CWC/2022/en.md:19]:
```suggestion
| Qualifiers | 2022-05-21/2022-05-22 |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**